enterprise/server/workflow/service/service.go

```diff
@@ -1254,6 +1255,19 @@ func (ws *workflowService) createActionForWorkflow(ctx context.Context, wf *tabl
 			},
 		},
 	}
+
+	if enableRunnerRecycling {
+		// Reuse the container/VM for the CI runner across executions if
+		// possible, and also keep the git repo around so it doesn't need to be
+		// re-cloned each time.
+		cmd.Platform.Properties = append(cmd.Platform.Properties, []*repb.Platform_Property{
+			{Name: "recycle-runner", Value: "true"},
+			{Name: "runner-recycling-max-wait", Value: (*workflowsRunnerMaxWait).String()},
+			{Name: "preserve-workspace", Value: "true"},
+			{Name: "use-self-hosted-executors", Value: useSelfHostedExecutors},
```

Comment:
Do we want to maintain the self-hosted-executor prop regardless?

Reply:
whoops, yeah, fixing.

Reply:
Done.